### ISSUE TITLE
Fix Post Navigation Widget

### ISSUE THREAD

**Issue by PaulPopravkin** [elementor/elementor#14252]
Remove ```float: right``` in ```.elementor-post-navigation .elementor-post-navigation__next a``` please. The right button width is not 100%.

I solved this problem like this:
``` 
selector .elementor-post-navigation .elementor-post-navigation__next a {
    float: none;
    display: flex;
    justify-content: flex-end;
} 
```

**Comment by shilo-ey** [comment]
Can you please share a screenshot or an explanation of the issue you are trying to solve?

Thanks

**Comment by PaulPopravkin** [comment]
@shilo-ey of course.
![1615993110957](https://user-images.githubusercontent.com/33669115/111488643-1a135d00-8742-11eb-8854-2ea62c195c5d.png)

**Comment by DorShahar** [comment]
Thanks for reporting @PaulPopravkin 
We will take care of this.

**Comment by avivu** [comment]
Hello,
This issue has been resolved in Elementor Pro v3.6.6

Feel free to update

Thanks!
